third_party/proto/cosmos/base/v1beta1/coin.proto

```diff
@@ -2,8 +2,6 @@ syntax = "proto3";
 package cosmos.base.v1beta1;
 
 import "gogoproto/gogo.proto";
-import "cosmos_proto/cosmos.proto";
-import "amino/amino.proto";
```

Comment:
I don't fully understand the changes being made in the proto files but I'm guessing it has to do with regen's `cosmos-proto`?

For reference I snagged these from https://buf.build/cosmos/cosmos-sdk/file/main:cosmos/base/v1beta1/coin.proto#L16 but I'm guessing this is perhaps an SDK 0.47.0 thing?